Makefile

```diff
@@ -0,0 +1,112 @@
+SHELL = /bin/bash
```

Comment:
Should this be `SHELL = /usr/bin/env bash` ?

Reply:
Maybe - I never really care that much. Re-reading https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my, your suggestion might be better for portability. Did you ever encounter issues with scripts using just `/bin/bash`?

Reply:
Personally, not with Bash. I have mainly seen this useful when using other interpreters, such as ruby or groovy.